CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
```

Comment:
The 'Salary' property appears before the required 'CompanyId' foreign key. This disrupts the logical grouping of related properties, which may cause confusion. Move this property below `CompanyId` for consistency and maintainable code layout.

```suggestion
public decimal Salary { get; set; }
```